packages/jest-runner/src/run_test.js

```diff
@@ -90,6 +98,7 @@ async function runTestInternal(
   } else if (globalConfig.verbose) {
     testConsole = new Console(consoleOut, process.stderr, consoleFormatter);
   } else {
+    // TODO: Should `BufferedConsole` receive `consoleFormatter` as well?
```

Comment:
@rickhanlonii I think so - the internal `_log` call in `BufferedConsole` is without sourcemaps now